pymongo/collection.py

```diff
@@ -795,6 +798,8 @@ def update_one(self, filter, update, upsert=False,
             MongoDB 4.2 and above.
           - `session` (optional): a
             :class:`~pymongo.client_session.ClientSession`.
+          - `let` (optional): Specifies a document with a list of variables
+            that can then be accessed using the form `$$<variable_name>`.
```

Comment:
Hmm my first impression is that this is very vague. Are there any docs that we can link to? Or any documentation guidance in the CRUD spec?

Reply:
Can you update this with the docs from the CRUD spec?:
```
Map of parameter names and values. Values must be constant or closed
expressions that do not reference document fields. Parameters can then be
accessed as variables in an aggregate expression context (e.g. "$$var").
```
I think this version more clearly describes the api.

Reply:
Done.